pages/form/returning-members.js

```diff
@@ -1,17 +1,21 @@
-import React, { useState, useContext, useEffect, useRef } from 'react';
-import { isEmail } from 'validator';
+import React, { useState, useContext } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useRouter } from "next/router";
 import { ThemeContext } from 'styled-components';
-import PageTemplate from '../frontend/components/templates/PageTemplate';
+import PageTemplate from '../../frontend/components/templates/PageTemplate';
 import {
-  SystemComponent,
-  SystemSpan,
-} from '../frontend/components/atoms/SystemComponents';
-import Subtitle from '../frontend/components/atoms/Subtitle';
-import Card from '../frontend/components/atoms/Card';
-import FieldSection from '../frontend/components/molecules/Form/FieldSection';
-import Button from '../frontend/components/atoms/Button';
+  SystemComponent
+} from '../../frontend/components/atoms/SystemComponents';
+import Card from '../../frontend/components/atoms/Card';
+import FieldSection from '../../frontend/components/molecules/Form/FieldSection';
+import FormHeader from '../../frontend/components/molecules/Form/FormHeader';
+import FormFooter from '../../frontend/components/molecules/Form/FormFooter';
 
-const SCHOOL_TERM_LABELS = [
+import {validateField, clearErrorMessages, isInvalidPhoneNumber, isInvalidStudentId} from '../../frontend/util'
+
+import { updateUser } from "../../frontend/store/reducers/userReducer";
+
+const SCHOOL_TERMS = [
   '1A Co-op',
   '1B Study',
   '1B Co-op',
```

Comment:
Sorry for commenting here, GH won't let me comment below because there's not git diff.

I have some concerns with hard-coding `PREV_TERMS` and `FUTURE_TERMS`. First of all, manually hard-coding these in every term will be a pain in the ass. But tbh, I don't think this is something that should be manually inputted by the user as a part of the form at all. I think that past terms is something that we should keep a record of through tracking in which term they're filling out the returning members form (we can capture the date when the user is filling it out and map that to the respective term). And now that I think of it, I don't really see the utility in keeping track of future terms they will be back, because it's possible they don't even end up returning.

Reply:
I totally agree with your suggestion for previous terms. This, however, will be a non-trivial change. I've created a ticket to track this proposed change, which could be handled next term: https://app.clickup.com/t/24x07yg

Future terms was always included in previous BOT/Returning Members forms. I will need to confirm with Team Directors why they included this Form Question. If this Question is no longer needed, we can certainly delete this altogether.
The proposed changes for Future Terms Form Question is also being tracked in the same ticket: https://app.clickup.com/t/24x07yg

Reply:
Sounds great, if you would like I could just send a message to the PMs asking what they think. Do you think there would need to be a lot of discussion?

Reply:
You can ask the PMs if you want.

I don't really know who created this form initially, so I might ask one of the former directors (Emrys, or Jennifer Chen) why they included those questions on the form.